benchmarks.go

```diff
@@ -454,6 +455,15 @@ var benchmarkCases = []BenchmarkCase{
 		UnsafeStringUnmarshal: true,
 		TimeSupport:           TSNoSupport,
 		APIKind:               AKManual,
+	}, {
+		Name: "idr",
+		URL:  "github.com/chmike/ditp",
+		New:  idr.NewIDRSerializer,
+
+		BufferReuseMarshal: true,
```

Comment:
Given recent discussions which introduced this, I'd suggest adding both with and without buffer reuse versions of the benchmark.